xml/admin_operating_pools.xml

```diff
@@ -486,31 +486,12 @@ pool4                     1.1 MiB      13      0     39                  0
    <para>
     To set a value to a pool, execute:
    </para>
-<screen>&prompt.cephuser;ceph osd pool set <replaceable>pool-name</replaceable> <replaceable>key</replaceable> <replaceable>value</replaceable></screen>
+<screen>&prompt.cephuser;ceph osd pool set <replaceable>pool-name</replaceable> <replaceable>KEY</replaceable> <replaceable>VALUE</replaceable></screen>
```

Comment:
Should this be wrapped for the PDF?

Reply:
i dont thing it goes beyond 80 characters if you make replaceables disappear

Reply:
Ah, fair :)